format/errors.js

```diff
@@ -0,0 +1,20 @@
+const { format } = require('logform');
+
+/*
+ * function errors (info)
+ * Handles special 'err' field and formats error according to ECS
+ */
+module.exports = format((info) => {
+  if (!Object.prototype.hasOwnProperty.call(info, 'err')) return info;
```

Comment:
I might be wrong but I'm pretty sure you can simply use `if (!info.hasOwnProperty('err')) return info;`. @v0vs ?

Reply:
eslint will complain:https://eslint.org/docs/rules/no-prototype-builtins

Reply:
yea it's pretty horrible, I agree.

Reply:
`Object.keys(info).includes('err')`?

Reply:
Whaaat - it's an amazing language)! So dynamic:)

Reply:
lol, I don't even have context. Don't know if that's true or not. Just threw it in.